ast.go

```diff
@@ -1058,7 +1094,22 @@ func (node *ColumnDef) String() string {
 		}
 		constraint = " " + strings.Join(constraints, " ")
 	}
-	return fmt.Sprintf("%s %s%s", node.Name, node.Type, constraint)
+	return fmt.Sprintf("%s %s%s", node.Column, node.Type, constraint)
+}
+
+// HasPrimaryKey checks if column definition has a primary key constraint.
+func (node *ColumnDef) HasPrimaryKey() bool {
+	if len(node.Constraints) == 0 {
+		return false
+	}
```

Comment:
Nit: I think we could remove this `if`. If the length is zero, L1106 will jump directly to L1112.